src/vs/workbench/api/browser/mainThreadTask.ts

```diff
@@ -491,6 +495,9 @@ export class MainThreadTask implements MainThreadTaskShape {
 					dto.name = ((dto.name === undefined) ? '' : dto.name); // Using an empty name causes the name to default to the one given by the provider.
 					return Promise.resolve(this._proxy.$resolveTask(handle, dto)).then(resolvedTask => {
 						if (resolvedTask) {
+							//TODO: @meganrogge figure out why this is needed and fix / remove
```

Comment:
@alexr00 any thoughts on this? I'm doing what you do with `_id`, in `$resolveTask`, but it's not working 🤔